examples/regression/4-tutorials/plot_main-tutorial-regression.py

```diff
@@ -1,32 +1,25 @@
-"""
+r"""
 ===============================
 Tutorial for tabular regression
 ===============================
+
+In this tutorial, we compare the prediction intervals estimated by MAPIE on a
+simple, one-dimensional, ground truth function :math:`f(x) = x \times \sin(x)`.
+Throughout this tutorial, we will answer the following questions:
+- How well do the MAPIE strategies capture the aleatoric uncertainty
```

Comment:
Add a line break here:
```suggestion
Throughout this tutorial, we will answer the following questions:

- How well do the MAPIE strategies capture the aleatoric uncertainty
```